### PR TITLE
Comment about mTLS in ingst architecture for proxy.

### PR DESCRIPTION
**Pull request by nimarezainia**
in reference to this issue: https://github.com/elastic/ingest-docs/issues/69

### CODE REVIEW

**Comment by nimarezainia** [comment]
@karenzone hope this is ok. Let me know.

File: docs/en/ingest-arch/15-proxy.asciidoc [hunk 1]
```diff
@@ -15,6 +15,8 @@ Use when::
 * Network restrictions prevent connection between {agent} network and network where {fleet-server} and {stack} are deployed, as when {fleet-server} is deployed on {ecloud}
 * Network restrictions prevent connection between {agent} and {fleet-server} network and the network where {stack} is deployed, as when {stack} is deployed on {ecloud}
 
+Currently {agent} is not able to present a certificate for connectivity to {fleet-server}. Therefore if a proxy placed between the {agent} and {fleet-server} is configured for mutual TLS, {agent]s won't be able
```
Comment (kilfoyle) [inline docs/en/ingest-arch/15-proxy.asciidoc:18]:
```suggestion
Currently {agent} is not able to present a certificate for connectivity to {fleet-server}. Therefore if a proxy placed between the {agent} and {fleet-server} is configured for mutual TLS, {agents} won't be able
```

File: docs/en/ingest-arch/15-proxy.asciidoc [hunk 2]
```diff
@@ -15,6 +15,8 @@ Use when::
 * Network restrictions prevent connection between {agent} network and network where {fleet-server} and {stack} are deployed, as when {fleet-server} is deployed on {ecloud}
 * Network restrictions prevent connection between {agent} and {fleet-server} network and the network where {stack} is deployed, as when {stack} is deployed on {ecloud}
 
+Currently {agent} is not able to present a certificate for connectivity to {fleet-server}. Therefore if a proxy placed between the {agent} and {fleet-server} is configured for mutual TLS, {agents} won't be able
+to establish connectivity to the {fleet server}.
```
Comment (kilfoyle) [inline docs/en/ingest-arch/15-proxy.asciidoc:19]:
```suggestion
to establish connectivity to {fleet server}.
```

**Comment by kilfoyle** [comment]
@karenzone Looks like this one slipped under our radar. I'll merge but if we need any changes just let me know.

**Comment by karenzone** [comment]
@nimarezainia Thank you for moving this one along. I appreciate the content.  Please assign me as a reviewer for quicker turn-around.  :-)

**Pull request merged**